packages/atomic/src/components/facets/facet-date-input/facet-date-input.tsx

```diff
@@ -93,13 +93,14 @@ export class FacetDateInput {
           ref={(ref) => (this.startRef = ref!)}
           class={inputClasses}
           aria-label={startAria}
+          placeholder="yyyy-mm-dd"
           required
           // API/Index minimum supported date
           min={this.formattedDateValue('1401-01-01')}
           max={this.formattedDateValue(this.end)}
           value={this.formattedDateValue(this.filterState.range?.start)}
           onInput={(e) =>
-            (this.start = (e.target as HTMLInputElement).valueAsDate!)
+            (this.start = new Date((e.target as HTMLInputElement).value))
```

Comment:
dayjs could perhaps be used instead of new Date (if the format is correct, doesn't make much a difference)

Reply:
What would happen if entering a different format? Would any code throw?

Also, are users alerted somehow that they are using an unsupported format? e.g. the apply button becomes disabled, error message etc.

Reply:
No code throes, simple a log in the console. I added a pattern which will indicate that the format is incorrect, it acts as a fallback. Otherwise we should stick to the Html validation that the browsers provides in most browsers